node/bft/tests/components/mod.rs

```diff
@@ -42,7 +42,7 @@ pub fn sample_ledger(
         committee.members().iter().map(|(address, (amount, _))| (*address, (*address, *address, *amount))).collect();
     let gen_key = *accounts[0].private_key();
     let public_balance_per_validator =
-        (1_500_000_000_000_000 - (num_nodes as u64) * 1_000_000_000_000) / (num_nodes as u64);
+        (<CurrentNetwork as Network>::STARTING_SUPPLY - (num_nodes as u64) * MIN_VALIDATOR_STAKE) / (num_nodes as u64);
```

Comment:
You should be able to write:
```suggestion
        (CurrentNetwork::STARTING_SUPPLY - (num_nodes as u64) * MIN_VALIDATOR_STAKE) / (num_nodes as u64);
```

Reply:
Good catch - [777c00d](https://github.com/AleoHQ/snarkOS/pull/3174/commits/777c00d51d70bac6d306c243becf23cb0d7a845a)